Failure of quality test

Recently flak8 quality tests, here is a recent error message

```
Run tox -e project,flake8
[12](https://github.com/ICB-DCM/pyPESTO/actions/runs/3927917745/jobs/6714984340#step:7:13)
project: install_deps> python -I -m pip install pyroma
[13](https://github.com/ICB-DCM/pyPESTO/actions/runs/3927917745/jobs/6714984340#step:7:14)
project: freeze> python -m pip freeze --all
[14](https://github.com/ICB-DCM/pyPESTO/actions/runs/3927917745/jobs/6714984340#step:7:15)
project: build==0.10.0,certifi==2022.12.7,charset-normalizer==3.0.1,docutils==0.19,idna==3.4,packaging==23.0,pep517==0.13.0,pip==22.3.1,Pygments==2.14.0,pyproject_hooks==1.0.0,pyroma==4.1,requests==2.28.2,setuptools==65.6.3,tomli==2.0.1,trove-classifiers==2023.1.12,urllib3==1.26.14,wheel==0.38.4
[15](https://github.com/ICB-DCM/pyPESTO/actions/runs/3927917745/jobs/6714984340#step:7:16)
project: commands[0]> pyroma --min=10 .
[16](https://github.com/ICB-DCM/pyPESTO/actions/runs/3927917745/jobs/6714984340#step:7:17)
------------------------------
[17](https://github.com/ICB-DCM/pyPESTO/actions/runs/3927917745/jobs/6714984340#step:7:18)
Checking .
[18](https://github.com/ICB-DCM/pyPESTO/actions/runs/3927917745/jobs/6714984340#step:7:19)
Getting metadata for wheel...
[19](https://github.com/ICB-DCM/pyPESTO/actions/runs/3927917745/jobs/6714984340#step:7:20)
Found pypesto
[20](https://github.com/ICB-DCM/pyPESTO/actions/runs/3927917745/jobs/6714984340#step:7:21)
------------------------------
[21](https://github.com/ICB-DCM/pyPESTO/actions/runs/3927917745/jobs/6714984340#step:7:22)
Final rating: 10/10
[22](https://github.com/ICB-DCM/pyPESTO/actions/runs/3927917745/jobs/6714984340#step:7:23)
Your cheese is so fresh most people think it's a cream: Mascarpone
[23](https://github.com/ICB-DCM/pyPESTO/actions/runs/3927917745/jobs/6714984340#step:7:24)
------------------------------
[24](https://github.com/ICB-DCM/pyPESTO/actions/runs/3927917745/jobs/6714984340#step:7:25)
project: OK ✔ in 6.54 seconds
[25](https://github.com/ICB-DCM/pyPESTO/actions/runs/3927917745/jobs/6714984340#step:7:26)
flake8: install_deps> python -I -m pip install 'flake8-bandit>=4.1.1' 'flake8-black>=0.3.3' 'flake8-bugbear>=22.8.23' 'flake8-colors>=0.1.9' 'flake8-comprehensions>=3.10.0' 'flake8-docstrings>=1.6.0' 'flake8-isort>=4.0.2' 'flake8-print>=5.0.0' 'flake8>=5.0.4'
[26](https://github.com/ICB-DCM/pyPESTO/actions/runs/3927917745/jobs/6714984340#step:7:27)
flake8: freeze> python -m pip freeze --all
[27](https://github.com/ICB-DCM/pyPESTO/actions/runs/3927917745/jobs/6714984340#step:7:28)
flake8: attrs==22.2.0,bandit==1.7.4,black==22.12.0,click==8.1.3,flake8==6.0.0,flake8-bandit==4.1.1,flake8-black==0.3.6,flake8-bugbear==23.1.14,flake8-colors==0.1.9,flake8-comprehensions==3.10.1,flake8-docstrings==1.6.0,flake8-isort==6.0.0,flake8-print==5.0.0,gitdb==4.0.10,GitPython==3.1.30,isort==5.11.4,mccabe==0.7.0,mypy-extensions==0.4.3,pathspec==0.10.3,pbr==5.11.1,pip==22.3.1,platformdirs==2.6.2,pycodestyle==2.10.0,pydocstyle==6.2.3,pyflakes==3.0.1,PyYAML==6.0,setuptools==65.6.3,smmap==5.0.0,snowballstemmer==2.2.0,stevedore==4.1.1,tomli==2.0.1,wheel==0.38.4
[28](https://github.com/ICB-DCM/pyPESTO/actions/runs/3927917745/jobs/6714984340#step:7:29)
flake8: commands[0]> flake8 pypesto test
[29](https://github.com/ICB-DCM/pyPESTO/actions/runs/3927917745/jobs/6714984340#step:7:30)
pypesto/ensemble/util.py:123:13: B028 'SAMPLE' is manually surrounded by quotes, consider using the `!r` conversion flag.
[30](https://github.com/ICB-DCM/pyPESTO/actions/runs/3927917745/jobs/6714984340#step:7:31)
pypesto/ensemble/util.py:123:13: B028 'OPTIMIZE' is manually surrounded by quotes, consider using the `!r` conversion flag.
[31](https://github.com/ICB-DCM/pyPESTO/actions/runs/3927917745/jobs/6714984340#step:7:32)
pypesto/history/hdf5.py:82:17: B028 'self.id' is manually surrounded by quotes, consider using the `!r` conversion flag.
[32](https://github.com/ICB-DCM/pyPESTO/actions/runs/3927917745/jobs/6714984340#step:7:33)
pypesto/history/hdf5.py:82:17: B028 'self.file' is manually surrounded by quotes, consider using the `!r` conversion flag.
[33](https://github.com/ICB-DCM/pyPESTO/actions/runs/3927917745/jobs/6714984340#step:7:34)
pypesto/optimize/optimizer.py:37:13: B028 'optimizer' is manually surrounded by quotes, consider using the `!r` conversion flag.
[34](https://github.com/ICB-DCM/pyPESTO/actions/runs/3927917745/jobs/6714984340#step:7:35)
pypesto/optimize/optimizer.py:993:17: B028 'method' is manually surrounded by quotes, consider using the `!r` conversion flag.
[35](https://github.com/ICB-DCM/pyPESTO/actions/runs/3927917745/jobs/6714984340#step:7:36)
pypesto/optimize/optimizer.py:1042:17: B028 'method' is manually surrounded by quotes, consider using the `!r` conversion flag.
[36](https://github.com/ICB-DCM/pyPESTO/actions/runs/3927917745/jobs/6714984340#step:7:37)
pypesto/optimize/optimizer.py:1050:17: B028 'local_method' is manually surrounded by quotes, consider using the `!r` conversion flag.
[37](https://github.com/ICB-DCM/pyPESTO/actions/runs/3927917745/jobs/6714984340#step:7:38)
pypesto/optimize/optimizer.py:1099:25: B028 'option' is manually surrounded by quotes, consider using the `!r` conversion flag.
[38](https://github.com/ICB-DCM/pyPESTO/actions/runs/3927917745/jobs/6714984340#step:7:39)
pypesto/sample/sampler.py:16:13: B028 'sampler' is manually surrounded by quotes, consider using the `!r` conversion flag.
[39](https://github.com/ICB-DCM/pyPESTO/actions/runs/3927917745/jobs/6714984340#step:7:40)
flake8: exit 1 (12.27 seconds) /home/runner/work/pyPESTO/pyPESTO> flake8 pypesto test pid=1792
[40](https://github.com/ICB-DCM/pyPESTO/actions/runs/3927917745/jobs/6714984340#step:7:41)
  project: OK (6.54=setup[5.09]+cmd[1.44] seconds)
[41](https://github.com/ICB-DCM/pyPESTO/actions/runs/3927917745/jobs/6714984340#step:7:42)
  flake8: FAIL code 1 (17.84=setup[5.57]+cmd[12.27] seconds)
[42](https://github.com/ICB-DCM/pyPESTO/actions/runs/3927917745/jobs/6714984340#step:7:43)
  evaluation failed :( (24.63 seconds)
[43](https://github.com/ICB-DCM/pyPESTO/actions/runs/3927917745/jobs/6714984340#step:7:44)
Error: Process completed with exit code 255.
```

--- a/.flake8
+++ b/.flake8
@@ -19,6 +19,8 @@ extend-ignore =
   D107
   # Empty method in abstract base class
   B027
+  # Using f"{foo!r}" instead of f"'{foo}'" (only until April 2023)
+  B028
 
 per-file-ignores =
     # Imported but unused

